Test-№1.1-from-19.10.2022/task3.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stddef.h>
+#include <string.h>
+#define maxSize 500
+
+int countOfSymbols(char *path)
+{
+    FILE *testFile = fopen(path, "r");
+    char symbol = "c";
+    if (testFile == NULL)
+    {
+        printf("This file does not exist!");
+        return -1;
+    }
+    printf("Your file read correctly!\n");
+    int bigArray[maxSize] = {0};
+    while (fscanf(testFile, "%c", &symbol) != EOF)
+    {
+        ++bigArray[symbol];
+    }
+    for (int i = 0; i < maxSize; ++i)
+    {
+        if (bigArray[i] != '0')

```

Comment:
```suggestion
        if (bigArray[i] > 0)

```
Видимо, предполагалось так.